plugins/modules/win_acl.ps1

```diff
@@ -206,7 +239,7 @@ Try {
     ElseIf ($state -eq "absent" -And $match -eq $true) {
         Try {
             $objACL.RemoveAccessRule($objACE)
-            If ($path_item.PSProvider.Name -eq "Registry") {
+            If (($path_item.PSProvider.Name -eq "Registry") -or ($path_item.PSProvider.Name -eq "ActiveDirectory")) {
```

Comment:
The handling for ActiveDirectory is also needed on line 227 or the module only works for the `absent` state in my testing